app/models/ta_membership.rb

```diff
@@ -4,12 +4,12 @@ class TaMembership < Membership
   after_create   { Repository.get_class.update_permissions }
   after_destroy  { Repository.get_class.update_permissions }
 
- def must_be_a_ta
-   if user && !user.is_a?(Ta)
-     errors.add('base', 'User must be a ta')
-     false
-   end
- end
+  def must_be_a_ta
```

Comment:
This is a good style change, but doesn't belong in this pull request. Please revert this change here, and then make a separate branch that pull request to fix the indentation for this function.

Reply:
Please remember to fix this.